### ISSUE TITLE
Additional feature request

### ISSUE THREAD

**Issue by yannicklung** [brandonbeecroft/Lorem-Ipsum-Plugin-for-Sketch#1]
Thanks! I love it so far! 
It is really useful to me but are you able to add a feature where I can define how many words should be displayed?

**Comment by lucaorio** [comment]
:+1: +1

**Comment by psteinweber** [comment]
Yes, that would be helpful. Emmet has a really awesome way for doing that: http://docs.emmet.io/abbreviations/lorem-ipsum/
